dreamhome.py

```diff
@@ -1,16 +1,15 @@
 from flask import Flask
 from database.db import initialize_db
+from api.routes.api_routes import api
+app = Flask(__name__, template_folder = 'api/views')
 
-app = Flask(__name__)
-
-app.config['MONGODB_SETTINGS'] = {
-    'host': 'mongodb://localhost/dreamhome'
-}
+app.config.from_envvar('ENV_FILE_LOCATION')
```

Comment:
I like that we have encapsulated our env variable data into a .gitignored location. This allows us to have "secret" information in the future